### ISSUE TITLE
Got "Undefined index: operator"

### ISSUE THREAD

**Issue by choirool** [GeneaLabs/laravel-model-caching#133]
# Issue 
Got  **"Undefined index: operator"** when use whereNotIn and whereIn

## Environment
Laravel Version: *5.5.34*
Laravel Model Caching Package Version: *0.2.21*
PHP Version: *7.2.3*
Operating System & Version: *Ubuntu 16.04.4*

## Stack Trace
```
Category::select('name', 'id')
                ->orderBy('name')
                ->whereNotIn('slug', ['foo', 'bar'])
                ->whereIn('id',[1,2,3])
                ->get();
```

**Comment by mikebronner** [comment]
@choirool Thanks for reporting!! I will take a look at this tomorrow. :)

**Comment by mikebronner** [comment]
@choirool In the mean time, can you update the package to the latest version and check if the error persists?

**Comment by mikebronner** [comment]
This should now be fixed in version 0.2.59. Could you give that a try and let me know how it works? Thanks again for reporting this!
